Gemfile

```diff
@@ -3,6 +3,9 @@ source 'http://rubygems.org'
 gem 'github-pages'
 gem 'jekyll-paginate'
 # gem 'redcarpet'
-
-# Added at 2018-01-15 15:18:35 +0000 by root:
-gem "jekyll-pagination", "~> 0.0.4"
+
+# Added at 2018-01-15 15:18:35 +0000 by root:
+gem "jekyll-pagination", "~> 0.0.4"
+
+# Added at 2018-01-30 by JohnWeaver // Allows CircleCI to upload to S3
+gem 's3_website '

```

Comment:
i'm guessing this shouldn't have a space after the  name. would also be nice to lock it to a version number.

Reply:
@skinofstars You are correct on the space.
Current version is v3.4.0, I'm not sure where CircleCI gets gems from, am I safe to use that version?

Reply:
if you add the gem using the command `bundle add s3_website`, it'll both add the latest version and update the lock file

Reply:
I'm working from the web interface, not sure where you want the command to be run? Please advise?